util/generate_build_files.py

```diff
@@ -37,6 +37,7 @@
     ('mac', 'x86_64', 'macosx', [], 'S'),
     ('win', 'x86', 'win32n', ['-DOPENSSL_IA32_SSE2'], 'asm'),
     ('win', 'x86_64', 'nasm', [], 'asm'),
+    ('win', 'aarch64', 'win64', [], 'S'),
```

Comment:
Flagging this for Kaufman

Reply:
This just means that we will have to re-run `generate_build_files.py` to generate the files for this newly added architecture. I will do this in another PR when I add a new test for ensuring that things stay in sync.